cves/kernel/CVE-2019-13648.yml

```diff
@@ -341,9 +347,32 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: true
+  stacktrace_with_fix: true
+  note: | 
```

Comment:
Better to give a link (if possible) then C&P the stacktrace

Reply:
Done